contributor_docs/zh/steward_guidelines.md

```diff
@@ -78,83 +79,88 @@
 
 ## 讨论
-此类问题有一个简洁的模板("讨论")，只有在特定讨论不适用于其他三个现有模板或更适合于论坛或Discord时才使用。
+此类问题有一个简洁的模板("讨论")，只有在特定讨论不适用于其他三个现有模板或更适合于论坛或 Discord 时才使用。
 
-- 如果以讨论的形式打开了一个问题，但应该是一个错误报告，应用正确的标签并删除"讨论"标签。如果尚未包含有关错误的其他信息(遵循上述[错误报告](#bug-report))，还应要求作者提供更多信息。
-- 如果以讨论的形式打开了一个问题，但与源代码贡献或其他与Github存储库/贡献过程/贡献社区相关的问题无关(例如，讨论p5.js的草图使用的最佳投影仪类型)，应将其重定向到论坛或Discord，并关闭问题。
+- 如果以讨论的形式提出了一个问题，但其实应该是一个错误报告，应该使用正确的标签并删除"讨论"标签。同时也应向作者索取未包含在错误报告中的其他信息。
+- 如果以讨论的形式打开了一个问题，但与源代码贡献或其他与 Github 存储库/贡献过程/贡献社区相关的问题无关(例如，讨论 p5.js 的草图使用的最佳投影仪类型)，应将其重定向到论坛或 Discord，并关闭问题。
 - 如果适用，应向讨论问题添加其他标签，以进一步标识讨论的类型。
 
 ---
 
 # 拉取请求
-几乎所有对p5.js存储库的代码贡献都是通过拉取请求进行的，管理者和维护者可能具有对存储库的推送访问权限，但在贡献代码时仍然鼓励他们遵循相同的问题 > 拉取请求 > 审查流程。以下是审查拉取请求时可以采取的一些步骤。
+几乎所有对 p5.js 存储库的代码贡献都是通过拉取请求进行的，管理者和维护者可能具有对存储库的推送访问权限，但在贡献代码时仍然鼓励他们遵循相同的问题 > 拉取请求 > 审查流程。以下是审查拉取请求时可以采取的一些步骤。
 
 - 拉取请求模板可以在[此处](../.github/PULL_REQUEST_TEMPLATE.md)找到。
 - 几乎所有拉取请求必须先打开并讨论相关的问题，这意味着在任何管理者或维护者审查PR之前，必须首先按照相关[问题工作流程](#issues)进行操作。
 	- 唯一不适用此规则的情况是非常小的拼写错误修正，这不需要打开问题，任何具有合并访问权限的人都可以合并该修复，即使他们不是特定领域的管理者。
 	- 尽管存在这个例外，但在实践中我们只会在通常鼓励贡献者开启新问题的情况下应用它。换句话说，如果对于这个例外是否适用存在疑问，只需打开一个问题即可。
-- 如果拉取请求不能完全解决引用的问题，您可以编辑原始帖子，将"解决 #OOOO"改为"处理 #OOOO"，这样当合并此PR时，不会自动关闭原始问题。
+- 如果拉取请求不能完全解决引用的问题，您可以编辑原始帖子，将"解决 #OOOO"改为"处理 #OOOO"，这样当合并此 PR 时，不会自动关闭原始问题。
 
 ## 简单修复
 像小的拼写错误修复之类的简单修复可以由具有合并访问权限的任何人直接合并，只需在PR的"更改的文件"选项卡中快速检查，并确保自动化CI测试通过。
 
-## Bug修复
-1. Bug修复应由相关领域的管理者审查，最好是同意修复相关问题的相同人员。
-2. PR的"更改的文件"选项卡可用于初步审查修复是否与问题讨论中描述的一致。
-3. 尽可能在本地进行必要的测试。Github CLI可以帮助简化部分流程。（请参阅下面的[Tips＆Tricks](#tips-tricks)了解更多详情）。
+## Bug 修复
+1. Bug 修复应由相关领域的管理者审查，最好是同意修复相关问题的相同人员。
+2. PR 的"更改的文件"选项卡可用于初步审查修复是否与问题讨论中描述的一致。
+3. 尽可能在本地进行必要的测试。Github CLI 可以帮助简化部分流程。（请参阅下面的[Tips＆Tricks](#tips-tricks)了解更多详情）。
 	- 修复应足够解决原始问题。
 	- 修复不应更改任何现有行为，除非在原始问题中已经达成一致。
-	- 修复不应对p5.js的性能产生重大影响。
-	- 修复不应对p5.js的可访问性产生任何影响。
-	- 修复应使用现代标准的JavaScript编码。
+	- 修复不应对 p5.js 的性能产生重大影响。
+	- 修复不应对 p5.js 的可访问性产生任何影响。
+	- 修复应使用现代标准的 JavaScript 编码。
 	- 修复应通过所有自动化测试，并在相关的情况下包含新测试。
 4. 如果需要进行任何其他更改，应根据[此处](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)所述，在相关行添加行级注释。
-   - 还可以使用建议块来建议具体的更改。
+   - 还可以使用建议模块来建议具体的更改:\
+     ![这个是"更改建议"按钮，在 Github PR 提供代码建议的时候使用](../images/suggest-change.png)\
+     ![一个自带修改建议符的修改建议在围栏代码块里显示](../images/suggested-value-change.png)\
+     ![一个修改建议预览为 diff](../images/suggestion-preview.png)
    - 如果需要进行多个更改，而不是多次添加单行注释，请按照[此处](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)所述进行多行注释并请求更改。
-   - 如果行级注释仅用于澄清或讨论，并且尚未请求更改，则在上一步中选择“Request changes”而不是“Comment”。
-5. 一旦PR已经被审查，并且不需要任何其他更改，负责人可以选择“Approve”选项（而不是“Comment”或“Request changes”）将PR标记为“Approved”。然后，如果需要，他们可以要求另一个负责人或维护者进行进一步审查，或者如果他们具有合并访问权限，可以合并PR，或者维护者将合并已批准的PR。
-6. 应该调用@all-contributors机器人，将任何新贡献者添加到README.md文件的贡献者列表中。
+   - 如果行级注释仅用于澄清或讨论，则在上一步中选择 “Request changes” 而不是 “Comment”。
```

Comment:
原文：- If line comments are just for clarification or discussion, choose “Comment” instead of "Request changes":\
   
如果行级注释仅用于澄清或讨论，则在上一步中选择 “Request changes” 而不是 “Comment”。->  如果行级注释仅用于澄清或讨论，则在上一步中选择 “Comment” 而不是 “Request changes”。